Parser.js

```diff
@@ -1,21 +1,8 @@
 'use strict';
 
 const Utf8Stream = require('./utils/Utf8Stream');
-
-const patterns = {
-  value1: /^(?:[\"\{\[\]\-\d]|true\b|false\b|null\b|\s{1,256})/,
-  string: /^(?:[^\"\\]{1,256}|\\[bfnrt\"\\\/]|\\u[\da-fA-F]{4}|\")/,
-  key1: /^(?:[\"\}]|\s{1,256})/,
-  colon: /^(?:\:|\s{1,256})/,
-  comma: /^(?:[\,\]\}]|\s{1,256})/,
-  ws: /^\s{1,256}/,
-  numberStart: /^\d/,
-  numberDigit: /^\d{0,256}/,
-  numberFraction: /^[\.eE]/,
-  numberExponent: /^[eE]/,
-  numberExpSign: /^[-+]/
-};
 const MAX_PATTERN_SIZE = 16;
```

Comment:
this const name is now confusing (compared to DEFAULT_PATTERN_SIZE) but I couldn't find a better name